code/modules/mining/abandoned_crates.dm

```diff
@@ -149,6 +149,234 @@
+					new /obj/item/stack/sheet/mineral/uranium{amount = 15}(src)
+					new /obj/item/stack/sheet/mineral/plasma{amount = 15}(src)
+					new /obj/item/stack/sheet/mineral/gold{amount = 15}(src)
+				if("midstuff6")
+					new /obj/item/book/granter/spell/barnyard(src)
+				if("midstuff7")
+					new /obj/item/borg/upgrade/vtec(src)
+					new /obj/item/borg/upgrade/disablercooler(src)
+				if("midstuff8")
+					new /obj/item/book/granter/spell/knock(src)
+				if("midstuff9")
+					new /obj/item/book/granter/spell/summonitem(src)
+					new /obj/item/book/granter/spell/forcewall(src)
+		if(129)
+			new /obj/item/stack/sheet/mineral/snow{amount = 25}(src)
+			new /obj/item/toy/snowball(src)
+			new /obj/item/twohanded/spear(src)
+		if(130)
+			new /mob/living/simple_animal/bot/medbot(src)
+			new /mob/living/simple_animal/bot/floorbot(src)
+			new /mob/living/simple_animal/bot/cleanbot(src)
+		if(131)
+			new /obj/item/card/emagfake(src)
+			new /obj/item/ammo_box/magazine/toy/pistol/riot(src)
+			new /obj/item/gun/ballistic/automatic/toy/pistol/riot(src)
+			new /obj/item/toy/sword(src)
+			new /obj/item/toy/cards/deck/syndicate(src)
+			new /obj/item/toy/syndicateballoon(src)
+			new /obj/item/bedsheet/syndie(src)
+			new /obj/item/toy/plush/nukeplushie(src)
+			new /obj/item/toy/nuke(src)
+			new /obj/item/disk/nuclear/fake(src)
+		if(132)
+			new /obj/item/pneumatic_cannon/speargun(src)
+		if(133)
+			new /obj/item/dnainjector/elvismut(src)
+			new /obj/item/dnainjector/firemut(src)
+			new /obj/item/dnainjector/glow(src)
+			new /obj/item/dnainjector/spastic(src)
+			new /obj/item/dnainjector/epimut(src)
+			new /obj/item/handmirror(src)
+			new /obj/item/razor(src)
+		if(134 to 138)
+			for(var/stick in 7)
+				stick = pick(subtypesof(/obj/item/flashlight/glowstick))
+				new stick(src)
+		if(139 to 141)
+			new /obj/effect/spawner/lootdrop/two_percent_xeno_egg_spawner(src)//Two percent for a crate to spawn a two percent xeno spawner.
+			new /obj/item/clothing/suit/xenos(src)                            //This means it's about one percent or less to get a xeno egg from an abandoned crate, I don't know.
+			new /obj/item/toy/toy_xeno(src)                                   //I actually don't give a shit how much percent it has.
+			new /obj/item/clothing/head/xenos(src)                            //I'm not memorizing the multiplication table. I don't have a calculator and my ass is fat.
+		if(142 to 146)
+			new /mob/living/simple_animal/cluwne(src)
+			new /mob/living/simple_animal/cluwne(src)
+			new /mob/living/simple_animal/cluwne(src)
+			new /mob/living/simple_animal/cluwne(src)
+		if(147)
+			new /obj/item/clothing/head/helmet/swat/nanotrasen(src)
+			new /obj/item/clothing/under/rank/security/officer(src)
+			new /obj/item/clothing/suit/armor/vest(src)
+			new /obj/item/clothing/shoes/combat/swat(src)
+			new /obj/item/clothing/gloves/combat(src)
+		if(148)
+			new /obj/item/stack/telecrystal/five(src)
```

Comment:
Not a good idea, all of the Traitor and its items are balanced around them having only 20TC without getting more from another Traitor, this would also allow them to solo buy romerol.

Reply:
So you think that will encourage all the traitors go hunt down a random abandoned crate, open it up and find the crystals? (but instead they find 4 clowns and a cluwnify injector)